prince-of-versions/src/main/java/co/infinum/princeofversions/ApplicationConfigurationImpl.java

```diff
@@ -0,0 +1,45 @@
+package co.infinum.princeofversions;
+
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.os.Build;
+
+/**
+ * This class provides application's version name and minimum SDK version.
+ */
+class ApplicationConfigurationImpl implements ApplicationConfiguration {
+
+    /**
+     * Version name
+     */
+    private String version;
+
+    /**
+     * Minimum SDK version
+     */
+    private int minSdk;
+
+    /**
+     * Creates application configuration for provided {@link Context}.
+     *
+     * @param context instance of application context from where is version name read.
+     */
+    ApplicationConfigurationImpl(Context context) {
+        minSdk = Build.VERSION.SDK_INT;
```

Comment:
I think this is confusing. This is not a minimum SDK version, but rather the SDK version of the current device, right?

Reply:
Indeed, the name is incorrect. Thanks